cmd/cluster/core/create.go

```diff
@@ -597,6 +601,15 @@ func (opts *RawCreateOptions) Validate(ctx context.Context) (*ValidatedCreateOpt
 		} else if !apierrors.IsNotFound(err) {
 			return nil, fmt.Errorf("hostedcluster doesn't exist validation failed with error: %w", err)
 		}
+
+		// Validate multi-arch aspects
+		kc, err := hyperutil.GetKubeClientSet()
+		if err != nil {
+			return nil, err
+		}
+		if err = validateMgmtClusterAndNodePoolCPUArchitectures(ctx, opts, kc); err != nil {
```

Comment:
Better to scope errors rather than reuse the parent scope
```suggestion
		if err := validateMgmtClusterAndNodePoolCPUArchitectures(ctx, opts, kc); err != nil {
```